packages/jsapi-components/src/useGetItemIndexByValue.ts

```diff
@@ -30,9 +33,13 @@ export function useGetItemIndexByValue<TValue>({
     const column = table.findColumn(columnName);
     const columnValueType = tableUtils.getValueType(column.type);
 
-    const index = await table.seekRow(0, column, columnValueType, value);
-
-    return index === -1 ? null : index;
+    try {
+      const index = await table.seekRow(0, column, columnValueType, value);
+      return index === -1 ? null : index;
+    } catch (err) {
+      log.debug('Error seeking row', { column, value, columnValueType });
+      return null;
```

Comment:
Any reason not to still throw? Consumer should be handling promise rejection anyway, no?

Reply:
fixed